translations/localify/ui.json

```diff
@@ -1997,5 +1998,158 @@
+    "ファン数を増やしていくことで、ウマ娘のクラスアップができます": "By getting more fans, you can upgrade your Uma Musume class",
+    "ファン数はレースに出走することで増やすことができます": "You can get more fans by running in races",
+    "前 へ": "Previous",
+    "ゴールドシップ① You know what I'm\ntalking about!": "Gold Ship① You know what I'm\ntalking about!",
+    "ゴールドシップ① You know what I'mtalking about!": "Gold Ship① You know what I'm\ntalking about!",
+    "育成機能メンテナンス前の育成データのため、\n育成を再開できません\n育成データ削除を行ってください": "Unable to resume training for data from \nbefore training function maintenance. \nPlease delete training data.",
+    "育成機能メンテナンス前の育成データのため、育成を再開できません育成データ削除を行ってください": "Unable to resume training for data from \nbefore training function maintenance. \nPlease delete training data.",
+    "報 酬": "Rewards",
+    "メジロマックイーンのヒミツ①": "Mejiro McQueen's secret ①",
+    "人目": "Uma",
+    "2人目選択中": "2nd chosen",
+    "1人目選択中": "1st chosen",
+    "現在の選択をリセットして、継承ウマ娘をまとめて選択してもよろしいですか？": "Reset current selection and auto-select all Uma Musume?",
+    "    <color=#ff4400>レンタルする場合はマニーを消費します</color>": "    <color=#ff4400>Renting spends money</color>",
+    "    <color=#FF4400>レンタルする場合はマニーを消費します</color>": "    <color=#FF4400>Renting spends money</color>",
+    "継承ウマ娘1人目": "Parent Uma Musume 1",
+    "継承ウマ娘2人目": "Parent Uma Musume 2",
+    "スピードとスピード上限がアップする因子です": "Increases Speed aptitude",
+    "パワーとパワー上限がアップする因子です": "Increases Power aptitude",
+    "スタミナとスタミナ上限がアップする因子です": "Increases Stamina aptitude",
+    "賢さと賢さ上限がアップする因子です": "Increases Wisdom aptitude",
+    "テイオーのジンクス": "Teio's Jinx",
+    "わがままテイオーと思い出の景色": "Selfish Teio and the Memorable Scenery",
+    "カイチョーみたいな勝負服": "An Outfit like Kaichou",
+    "キャッチコピー": "Bio",
+    "思い出": "Memories",
+    "テイオー様にお任せあれ": "Leave it to Teio-sama",
+    "無敵のテイオー様！": "The invincible Teio-sama!",
+    "これからもよろしくねトレーナー！\n無敵のテイオー伝説には、\n絶対にキミが必要なんだからっ！": "Keep up the good work, Trainer! \nFor the legend of the invincible Teio, \nyou're absolutely necessary!",
+    "これからもよろしくねトレーナー！　無敵のテイオー伝説には、絶対にキミが必要なんだからっ！": "Keep up the good work, Trainer! \nFor the legend of the invincible Teio, \nyou're absolutely necessary!",
+    "ミホノブルボンのヒミツ①": "Mihono Bourbon's secret ①",
+    "やっほー、トウカイテイオーだよ！\n無敗の三冠ウマ娘になるボクのこと、\nぜーったい見逃さないでよねっ！": "Hey there, I'm Tokai Teio! \nI'm going to be the undefeated Triple Crown \nwinner, so don't miss out on me!",
+    "やっほー、トウカイテイオーだよ！　無敗の三冠ウマ娘になるボクのこと、ぜーったい見逃さないでよねっ！": "Hey there, I'm Tokai Teio! \nI'm going to be the undefeated Triple Crown \nwinner, so don't miss out on me!",
+    "親愛度ランク1で解放": "Unlocks at affection rank 1",
+    "親愛度ランク2で解放": "Unlocks at affection rank 2",
+    "親愛度ランク3で解放": "Unlocks at affection rank 3",
+    "親愛度ランク4で解放": "Unlocks at affection rank 4",
+    "親愛度ランク5で解放": "Unlocks at affection rank 5",
+    "[Beyond the Horizon]\nTokai Teioを獲得で解放": "Unlocks after acquiring \n[Beyond the Horizon] Tokai Teio",
+    "[Beyond the Horizon]Tokai Teioを獲得で解放": "Unlocks after acquiring \n[Beyond the Horizon] Tokai Teio",
```

Comment:
These must have a generic version to put in static.json somewhere.
The jp text isn't correct anyway as it's using the mdb translation. (it probably works cause of the id mapping but it's not good practice)